grails-app/controllers/org/pih/warehouse/partialReceiving/PartialReceivingController.groovy

```diff
@@ -10,8 +10,12 @@ class PartialReceivingController {
 		redirect(action: "create")
 	}
 
-	// This template is generated by webpack during application start
 	def create = {
+		Shipment shipment = Shipment.get(params.id)
```

Comment:
This should probably be the stock movement. I know it's basically the same thing, but just want it to be consistent in case we end up refactoring and removing shipment altogether. I can rewrite this the way I'd expect it to work.

Reply:
Ok, that's fine, so you can update this PR.

Reply:
It looks like the Partial Receiving feature was developed using the shipment ID so I guess there's not much refactoring I can do here at the moment.